graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/processor/IfConditionModelBuilder.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.graphql.schema.graphqlfirst.processor;
+
+import graphql.language.InputValueDefinition;
+import io.stargate.db.query.Predicate;
+import java.util.Map;
+
+class IfConditionModelBuilder extends ConditionModelBuilderBase {
+
+  IfConditionModelBuilder(
+      InputValueDefinition argument,
+      String operationName,
+      EntityModel entity,
+      Map<String, EntityModel> entities,
+      ProcessingContext context) {
+    super(context, argument, operationName, entity, entities);
+  }
+
+  @Override
+  protected String getDirectiveName() {
+    return "cql_if";
+  }
+
+  @Override
+  protected void validate(FieldModel field, Predicate predicate) throws SkipException {
+    // The CQL IF works only for regular columns (non PK, CK)
+    if (field.isPartitionKey()) {
+      invalidMapping(
+          "The cql_if is not supported for partition keys (field %s)", field.getGraphqlName());
```

Comment:
```suggestion
          "@cql_if is not supported for partition keys (field %s)", field.getGraphqlName());
```
To be consistent with other messages that refer to directives (e.g. `FieldModelBuilder`, line 213).